app/javascript/components/patient/Patient.js

```diff
@@ -238,6 +239,19 @@ class Patient extends React.Component {
               {this.renderEditLink('Contact Information', 2)}
             </div>
             <div className="item-group">
+              {this.props.details.date_of_birth && isMinor(this.props.details.date_of_birth) && (
+                <React.Fragment>
```

Comment:
so I think I like this better, thoughts?

![Screen Shot 2021-06-22 at 4 07 58 PM](https://user-images.githubusercontent.com/35042815/122992479-2c6dea00-d374-11eb-801c-c255da7f7ae8.png)

Reply:
this would also allow you to just remove the period

Reply:
Works for me!